docs/docs/rest-api/overview.mdx

```diff
@@ -1,15 +1,22 @@
 ---
-title: Rest API Overview
-subtitle: Learn how to use the MagicBell REST API
+title: API Overview
+subtitle: The API spec, error codes, rate limits, and other must-know information.
 ---
 
-This section describes the resources that make up the official MagicBell REST
-API. Use it to send notifications to your users and retrieve notifications
-previously sent to a user, change their state, delete them, etc.
+## The Basics
 
-The API is hosted at `https://api.magicbell.com`. It utilizes standard HTTP response codes and returns JSON responses.
+|                |                             |
+| -------------- | --------------------------- |
+| Endpoint       | `https://api.magicbell.com` |
+| `Accept`       | `application/json`          |
```

Comment:
Our table style for docs needs an update. Every other row is light text on a white background. 

<img width="968" alt="Screen Shot 2023-05-24 at 1 15 16 PM" src="https://github.com/magicbell-io/public/assets/106825828/39e9d4c4-76fe-4078-b7b3-534c591c928c">